desktop-app/src/renderer/components/ToolBar/index.tsx

```diff
@@ -6,17 +6,39 @@ import {
   setRotate,
 } from 'renderer/store/features/renderer';
 import { Icon } from '@iconify/react';
+import { useEffect, useRef } from 'react';
 import NavigationControls from './NavigationControls';
 import Menu from './Menu';
 import Button from '../Button';
 import AddressBar from './AddressBar';
 import ColorSchemeToggle from './ColorSchemeToggle';
 
+function useKey(key, cb) {
+  const callbackRef = useRef(cb);
+  useEffect(() => {
+    callbackRef.current = cb;
+  });
+
+  useEffect(() => {
+    function handle(event) {
+      if (event.code === key) {
+        callbackRef.current(event);
+      }
+    }
+    document.addEventListener('keypress', handle);
+    return () => document.removeEventListener('keypress', handle);
+  }, [key]);
+}
 const ToolBar = () => {
   const rotateDevice = useSelector(selectRotate);
   const isInspecting = useSelector(selectIsInspecting);
   const dispatch = useDispatch();
 
+  function handleInspectShortcut() {
+    dispatch(setIsInspecting(!isInspecting));
+  }
+  useKey('KeyS', handleInspectShortcut);
```

Comment:
Can you please validate once that this listener to `s` key is not hijacking user inputs to text boxes? 

On both the address bar and some input boxes in the previewed page as well.

Reply:
Also, is there a specific reason you chose `s` as the shortcut? 

I would prefer `i` for `inspect element`.